### PR TITLE
ISSUE-37: revoking access token on DELETE

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: protocols/jmap/src/test/java/org/apache/james/jmap/JMAPAuthenticationTest.java [hunk 1]
```diff
@@ -80,7 +81,7 @@ public void setup() throws Exception {
         mockedUsersRepository = mock(UsersRepository.class);
         mockedZonedDateTimeProvider = mock(ZonedDateTimeProvider.class);
         continuationTokenManager = new SignedContinuationTokenManager(new JamesSignatureHandlerProvider().provide(), mockedZonedDateTimeProvider);
-        accessTokenManager = new AccessTokenManagerImpl(new MemoryAccessTokenRepository(100));
+        accessTokenManager = new AccessTokenManagerImpl(new MemoryAccessTokenRepository(3600000));
```
Comment (aduprat) [inline protocols/jmap/src/test/java/org/apache/james/jmap/JMAPAuthenticationTest.java]: You may use something like that

``` java
TimeUnit.SECONDS.convert(1, TimeUnit.HOURS);
```

Comment (rouazana) [inline protocols/jmap/src/test/java/org/apache/james/jmap/JMAPAuthenticationTest.java]: ok

**Comment by mbaechler** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Comment by rouazana** [comment]
Merged on linagora/jmap (james-project repository)

**Pull request closed without merging**